ext/opentelemetry-ext-jaeger/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Released 2020-05-27
 
 - Transform resource to tags when exporting
   ([#645](https://github.com/open-telemetry/opentelemetry-python/pull/645))
+- ext/boto: Could not serialize attribute aws.region to tag when exporting via jaeger
```

Comment:
Would you mind changing this to be more descriptive of the change? I think the "why" is good too, but knowing precisely what the fix or behavioral change was will apply more broadly to those reading the changelog.

Reply:
ack